src/db/SQL.js

```diff
@@ -402,7 +402,7 @@ define([
 				return results;
 			}));
 			var store = this;
-			return new QueryResults(results, {
+			return QueryResults.default(results, {
```

Comment:
I realize `QueryResults` isn't a constructor per se, but we've always kind of treated it like one.  I realize this is awkward and will make a note about it on #137.